data/json/items/ammo/762.json

```diff
@@ -39,6 +39,7 @@
     "copy-from": "762_m43",
     "type": "AMMO",
     "name": "reloaded 7.62x39mm M43",
+    "description": "7.62x39mm ammunition with 121.9gr bullets.  Developed in WW2 by the Soviet Union the 7.62x39mm round rapidly became extremely popular all over the world.  The bullet has poor wounding potential due to its stability, only begining to yaw after 26cm.",
```

Comment:
"...only begining to yaw after 26cm." That should be **beginning** with two 'n's.

Reply:
Could you fix the original 762_m43 entry also while your at it.

Reply:
Fixed